Wrappers/Python/ccpi/viewer/CILViewerBase.py

```diff
@@ -0,0 +1,278 @@
+import vtk
+from ccpi.viewer.utils.io import SaveRenderToPNG
+
+SLICE_ORIENTATION_XY = 2  # Z
+SLICE_ORIENTATION_XZ = 1  # Y
+SLICE_ORIENTATION_YZ = 0  # X
+
+CONTROL_KEY = 8
+SHIFT_KEY = 4
+ALT_KEY = -128
+
+SLICE_ACTOR = 'slice_actor'
+OVERLAY_ACTOR = 'overlay_actor'
+HISTOGRAM_ACTOR = 'histogram_actor'
+HELP_ACTOR = 'help_actor'
+CURSOR_ACTOR = 'cursor_actor'
+CROSSHAIR_ACTOR = 'crosshair_actor'
+LINEPLOT_ACTOR = 'lineplot_actor'
+WIPE_ACTOR = 'wipe_actor'
```

Comment:
I guess these could be defined in `ccpi.viewer` rather than in `CILViewerBase`

Reply:
In the init file?

Reply:
I have moved to init file